packages/commonwealth/client/styles/components/component_kit/new_designs/cw_text_input.scss

```diff
@@ -0,0 +1,100 @@
+@import '../../../shared';
+
+@mixin iconPositionStyles {
+  position: absolute;
+  top: 50%;
+  transform: translateY(-50%);
+}
+
+.MessageRow {
+  display: flex;
+
+  &.hasFeedback {
+    align-items: flex-end;
+    gap: 8px;
+    justify-content: space-between;
+
+    > * {
+      flex: 1;
+    }
+  }
+
+  .feedback-message-text.Text {
+    justify-content: flex-end;
+    margin-bottom: 4px;
+    text-align: right;
+
+    &.success {
+      color: $green-600;
+    }
+
+    &.failure {
+      color: $rorange-500;
+    }
+  }
+}
+
+.TextInput {
```

Comment:
by doing this, you are overriding existing Input styles. It is intended behaviour? Do we want to just create new input in this PR or we want to Create and Replace it across the board? Right know it is messed up because Input from new_designs is used only in component showcase, however, the styles are overriding the old input what breaks the app in many places. cc @zakhap 
![image](https://github.com/hicommonwealth/commonwealth/assets/14819225/af79066e-5f36-4b41-a545-46029466cb65)

![image](https://github.com/hicommonwealth/commonwealth/assets/14819225/841b9aef-e14d-4596-97bf-d0033b1d6140)

![image](https://github.com/hicommonwealth/commonwealth/assets/14819225/661fb453-3c4f-457c-ad7b-e1b5a9094356)

Reply:
> 

I changed the old 'TextInput' class to 'OldTextInput', since it will probably not going to be used in the future.

@zakhap are we changing all TextInputs to the new design system one?

Reply:
Apparently changing it to OldTextInput is not enough. Some styles are still messed up here and there, because your inputs are not 100% in isolation. 

1. edit profile page: 
- vertical margin are messed up
- there is error but green text on validation text
- validation text is misaligned

![image](https://github.com/hicommonwealth/commonwealth/assets/14819225/732b90fd-6ae6-4413-acdc-ef76f3689ada)

2. create thread page:
- both inputs are full width in block styles

![image](https://github.com/hicommonwealth/commonwealth/assets/14819225/e5a55850-6275-49f0-ba18-0252a0a84856)

In both cases your changes are affecting the old styles which is not desired. Probably there are more places like this in the app where inputs are messed up

Reply:
> Apparently changing it to OldTextInput is not enough. Some styles are still messed up here and there, because your inputs are not 100% in isolation.
> 
> 1. edit profile page:
> 
> * vertical margin are messed up
> * there is error but green text on validation text
> * validation text is misaligned
> 
> ![image](https://user-images.githubusercontent.com/14819225/257487817-732b90fd-6ae6-4413-acdc-ef76f3689ada.png)
> 
> 2. create thread page:
> 
> * both inputs are full width in block styles
> 
> ![image](https://user-images.githubusercontent.com/14819225/257488277-e5a55850-6275-49f0-ba18-0252a0a84856.png)
> 
> In both cases your changes are affecting the old styles which is not desired. Probably there are more places like this in the app where inputs are messed up

1. edit profile page

- When you change the page to use the new component (importing from `**/component_kit/new_designs/CWTextInput`) the 'vertical margin' you mentioned probably depends on the isCompact props, which defines its class, either `large` or `small`.
- validation text and color fixed
- validation text was misaligned on old component. Working fine on new one.

![image](https://github.com/hicommonwealth/commonwealth/assets/30223098/45c40e31-aaea-4e3e-9452-31a4f6edb901)

2. create thread page

Same here: changing the component to new CWTextInput + using isCompact and fullWidth props make the two inputs appear as intended:

![image](https://github.com/hicommonwealth/commonwealth/assets/30223098/ae247f41-5982-4a63-bafe-6a8e4b6a60e4)

![image](https://github.com/hicommonwealth/commonwealth/assets/30223098/7d355c13-764f-439a-87ca-967b6185b493)